Resources/doc/reference/configuration.rst

```diff
@@ -173,10 +177,10 @@ Full Configuration Options
                     - bundles/sonatacore/vendor/jquery/dist/jquery.min.js
                     - bundles/sonataadmin/vendor/jquery.scrollTo/jquery.scrollTo.min.js
                     - bundles/sonatacore/vendor/moment/min/moment.min.js
-                    - bundles/sonatacore/vendor/bootstrap/dist/js/bootstrap.min.js
-                    - bundles/sonatacore/vendor/eonasdan-bootstrap-datetimepicker/build/js/bootstrap-datetimepicker.min.js
                     - bundles/sonataadmin/vendor/jqueryui/ui/minified/jquery-ui.min.js
                     - bundles/sonataadmin/vendor/jqueryui/ui/minified/i18n/jquery-ui-i18n.min.js
+                    - bundles/sonatacore/vendor/bootstrap/dist/js/bootstrap.min.js
```

Comment:
i know this is not an obvious change, but for future updating we should use the generated stuff as i it comes directly from the command